pymysql/connections.py

```diff
@@ -1100,6 +1131,11 @@ def _read_ok_packet(self, first_packet):
         self.message = ok_packet.message
         self.has_next = ok_packet.has_next
 
+    def _read_load_local_packet(self, first_packet):
+        load_packet = LoadLocalPacketWrapper(first_packet)
+        local_packet = LoadLocalFile(load_packet.filename, self.connection)
+        local_packet.send_data()
+
```

Comment:
ok_packet = self.connection._read_packet()
if not ok_packet.is_ok_packet():
    raise OperationalError(2014, "Commands Out of Sync")
self._read_ok_packet(ok_packet)